### PR TITLE
Search api w/ paths

### PR DESCRIPTION
**Pull request by gokhankici**

### CODE REVIEW

File: fetch-server/README.md [hunk 1]
```diff
@@ -11,3 +11,59 @@ Fetch Flask App
...
+``` shell
+wget http://download.osgeo.org/postgis/source/postgis-2.4.1.tar.gz
+cd postgis-2.4.1
+PATH="$HOME/postgres/bin:$PATH" ./configure
+PATH="$HOME/postgres/bin:$PATH" make
+PATH="$HOME/postgres/bin:$PATH" make install
+```
+
+```shell
+> ~/postgres/bin/psql fetch
```
Comment (soelgary) [inline fetch-server/README.md]: This should be `fetch_db`

**Review by soelgary — COMMENTED** [review]
Just one minor change to the readme, otherwise everything looks good and works

File: fetch-server/search.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    'is_water'       : fields.Boolean,
+    'is_garbage_can' : fields.Boolean,
+    'is_poop_bag'    : fields.Boolean,
+    'coodinates'     : Coordinates(attribute='path')
+}
+
+# filter & respond to the search query
+class RouteSearch(Resource):
+    @marshal_with(route_fields, envelope='results')
+    def get(self):
```
Comment (soelgary) [inline fetch-server/search.py:41]: This search is actually off. For some reason when I search `?is_poop_bag=False` the backend is converting it to True. It is doing the same with the wildlife endpoints that I am creating. Ignore it for now though until we know the fix

Comment (gokhankici) [reply fetch-server/search.py:41]: In order to make it false, I had to give it just `?is_poop_bag`.

**Pull request merged**